### DISCUSSION TITLE
Miscellanoeus

### DISCUSSION THREAD

**Discussion by IshaC13** [snigi-gupta/AttendingUniversityAtBuffalo#28] [General]
Hello!
Can you recommend a few places to shop for winter wear and other general requirements?

**Comment by snigi-gupta** [comment]
Hi @IshaC13 
Firstly I apologize for the delay in my response. Lately, I have been dealing with a lot of change, hence the delay.

Shop for winter wear from US itself. Don't bring much from your homeland country. If you do want to get something from home, I would recommend shopping at Decathalon. 
If you are already in US, then I would recommend shopping online (Amazon, H&M etc.) and for jackets and all go with Burlington, Columbia, LandsDown etc. UB GISA also takes students to a factory outlet so you can look and buy jackets, shoes, boots, mufflers, gloves etc. 

Let me know if you are looking for something in particular.

**Reply by IshaC13** [reply]
Thanks much. This helps :)

**Comment by abhiraj24** [comment]
Hi hope you doing well.

Just wanted some clarification in some of the questions I had.

1.) How much should we show as funds in our visa interview. I heard its 1.5 X of i20.

2.) Can you give a brief information about the academic calendar.?how are the courses basically span out in 1.5 years?

**Reply by snigi-gupta** [reply]
@abhiraj24 ,
1. Yes, sounds about right. Can also be more or less, its a personal preference.
2. I do not understand what you are trying to ask here. If you wish to know the no. credits to be pursued each sem, I think going through my CSE related gists/repos will be helpful. Else read the graduate handbook (link in my cse gists) and you can also google this stuff up.
